### PR TITLE
Add Hacker Laws

### PR DESCRIPTION
**Pull request by dwmkerr**
https://github.com/dwmkerr/hacker-laws/pull/164

This list has generated a reasonably large following and is regularly updated. I don't think there is yet something like it in the Awesome collection.

### By submitting this pull request I confirm I've read and complied with the below requirements

❌ 

It was suggested for me to add Hacker Laws to this list. I've reviewed one PR which was not yet reviewed. However; my repo does:

- Does not have the name `awesome-hacker-law` (just `hacker-laws`)
- Uses MIT
- And some other issues

It was not intended to be an awesome list. But happy to link to Awesome (via this PR https://github.com/dwmkerr/hacker-laws/pull/164) if you'd like to include it, also more than happy to close this PR if it seems my content is not relevant enough!

Cheers,

Dave

### CODE REVIEW

**Comment by caufieldjh** [comment]
I would suspect that this list is incompatible with Awesome, though it looks like a very useful reference in its own right.

File: readme.md [hunk 1]
```diff
@@ -467,7 +467,7 @@
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
 - [Product Management](https://github.com/dend/awesome-product-management#readme) - Learn how to be a better product manager.
 - [Roadmaps](https://github.com/liuchong/awesome-roadmaps#readme) - Gives you a clear route to improve your knowledge and skills.
-
+- [Hacker Laws](https://github.com/dwmkerr/hacker-laws) - Laws, theories, principles and patterns that developers and techies useful and come across in the industry.
```
Comment (cliffordfajardo) [inline readme.md]: Fix some minor grammar mistake, add a missing word before the word `useful`.

Perhaps something like:

```
Laws, theories, principles and patterns that developers and techies may find useful and come across in the industry"
```

**Comment by cliffordfajardo** [comment]
Make sure to add a comment to this pull request with the word `unicorn` as indicated by the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md).to let us know you've read everything:

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**